packages/openneuro-app/src/scripts/dataset/dataset.store.js

```diff
@@ -1277,21 +1277,30 @@ let datasetStore = Reflux.createStore({
    */
   revalidate() {
     let dataset = this.data.dataset
-    scitran.addTag('projects', dataset._id, 'validating').then(() => {
-      dataset.status.validating = true
-      this.update({ dataset })
-      crn.validate(dataset._id).then(res => {
-        let validation = res.body.validation
-        dataset.status.validating = false
-        dataset.validation = validation
-        dataset.summary = res.body.summary
-        dataset.status.invalid =
-          validation.errors &&
-          (validation.errors == 'Invalid' || validation.errors.length > 0)
+    scitran
+      .addTag('projects', dataset._id, 'validating')
+      .then(() => {
+        dataset.status.validating = true
         this.update({ dataset })
-        this.updateModified()
+        return crn.validate(dataset._id).then(res => {
+          let validation = res.body && res.body.validation
```

Comment:
this will cause validation to be a boolean, but it needs to be the object that sits at res.body.validation

let validation = (res.body && res.body.validation) ? res.body.validation : {}